src/js/components/auditlogs/auditlogslist.js

```diff
@@ -39,8 +40,12 @@ const fallbackFormatter = data => {
 
 const changeMap = {
   default: { component: 'div', actionFormatter: fallbackFormatter, title: 'defaultTitle' },
-  artifact: { actionFormatter: data => `uploaded ${decodeURIComponent(data.artifact.name)}`, component: ArtifactLink },
-  deployment: { actionFormatter: data => `to ${decodeURIComponent(data.deployment.name)}`, component: DeploymentLink },
+  artifact: { actionFormatter: data => `${decodeURIComponent(data.artifact.name)}`, component: ArtifactLink },
+  deployment: { actionFormatter: data => `${decodeURIComponent(data.deployment.name)}`, component: DeploymentLink },
+  deviceDecomissioned: { actionFormatter: data => `${decodeURIComponent(data.id)}`, component: 'div' },
+  deviceRejected: { actionFormatter: data => `${decodeURIComponent(data.id)}`, component: DeviceRejectedLink },
+  deviceGeneral: { actionFormatter: data => `${decodeURIComponent(data.id)}`, component: DeviceLink },
```

Comment:
`decodeURIComponent()` already produces a string, so if you don't combine the result with another string (as it was in `` `to ${decode...}` ``) it is better to omit the surrounding `` `${...}` `` 👍

Reply:
done, thanks!